include/cutlass/gemm/kernel/intel_pvc_gemm.hpp

```diff
@@ -133,133 +137,171 @@ class GemmUniversal<
   // Methods
   //
 
-  // Convert to underlying arguments. In this case, a simple copy for the aliased type.
-  static
-  Params
-  to_underlying_arguments(Arguments const& args, void* workspace) {
-    (void) workspace;
-    return {
-      args.mode,
-      args.problem_shape,
-      CollectiveMainloop::to_underlying_arguments(args.problem_shape, args.mainloop, workspace),
-      CollectiveEpilogue::to_underlying_arguments(args.problem_shape, args.epilogue, workspace)
-    };
+  // Convert to underlying arguments. In this case, a simple copy for the
+  // aliased type.
```

Comment:
I meant unnecessary reformatting of unchanged lines in general. Not just the copyright header. And of course if we reformat we need to follow the style guide.